7E3.json

```diff
@@ -0,0 +1,4473 @@
+{
```

Comment:
Some value origins are unknown / dummy. Can you fill them?
Interestingly, values with value origin **Unknown-/Dummy value not used in   the simulation** were in fact copied from a simulation :)

Name | Top Container | Organ | Compartment | Molecule | Value | Value Origin
-- | -- | -- | -- | -- | -- | --
Fraction   of blood for sampling | Organism | Bone |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Brain |   |   | 0 | Unknown
Lipophilicity |   |   |   | 7,00E+03 | -5.000 Log Units | Unknown-/Dummy value not used in   the simulation
Fraction   of blood for sampling | Organism | Fat |   |   | 0.180 | Unknown
Fraction   unbound (plasma, reference value) |   |   |   | 7,00E+03 | 1.000 | Unknown-Assumption
Fraction   of blood for sampling | Organism | Gonads |   |   | 0.180 | Unknown
Solubility   at reference pH |   |   |   | 7,00E+03 | 9999.000 mg/l | Unknown-/Dummy value not used in   the simulation
Fraction   of blood for sampling | Organism | Heart |   |   | 0.180 | Unknown
Reference   pH |   |   |   | 7,00E+03 | 7.000 | Unknown-/Dummy value not used in   the simulation
Fraction   of blood for sampling | Organism | Kidney |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Stomach |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Spleen |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Small Intestine |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Large Intestine |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Liver |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Lung |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Muscle |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Pancreas |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Skin |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Duodenum |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Upper Jejunum |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Lower Jejunum |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Upper Ileum |   |   | 0.180 | Unknown
Start   concentration of free FcRn (endosome) | Organism | Endogenous IgG Container | Endosome |   | 0 µmol/l | Unknown-FcRn knockout
Fraction   of blood for sampling | Organism | Lower Ileum |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Cecum |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Colon Ascendens |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Colon Transversum |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Colon Descendens / Distal Colon   1 |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Colon Sigmoid / Distal Colon 2 |   |   | 0.180 | Unknown
Fraction   of blood for sampling | Organism | Rectum |   |   | 0.180 | Unknown

Reply:
added for "Fraction of blood for sampling" parameters now. Parameters like lipophilicity or solubilty have to be set even for large molecules despite that they are effectively not used in the model. Therefore, I had added the respective comment before.